api/docs/http_api_index.md

```diff
@@ -95,6 +107,33 @@ error responses will uniformly be a JSON object with the following properties:
 
 A {file:api/schemas/error.json JSON schema for the error objects} is also available.
 
+CA V1 HTTP API
+--------------
+
+The CA API contains all of the endpoints used in support of Puppet's PKI
+system.
+
+The CA V1 endpoints share the same basic format as the Master V3 API, since
+they are also based off of puppet's internal "indirector". However, they have
+a different prefix and version. The endpoints thus follow the form:
+`/puppet-ca/v1/:indirection/:key?environent=:environment` where
+
+  * `:environment` is the name of the environment that should be in effect for
+    the request. Not all endpoints need an environment, but the path component
+    must always be specified.
```

Comment:
Same question here about whether to refer to it as a "path component" or as a query parameter or some such.